test/dynamo/test_dynamo_integrations_util.py

```diff
@@ -84,6 +84,18 @@ def test_get_graph_hash(self):
     xla_out_2 = xla_dummy_model(xla_input)
     assert (hash == torch_xla._XLAC._get_graph_hash([xla_out_2]))
 
+  def test_get_graph_hash_raises_error_on_non_xla_tensor(self):
+    tensors = [
+        torch.rand(10, device=torch_xla.device()),
+        torch.rand(10),
+        torch.rand(10, device=torch_xla.device()),
+    ]
+    error_message = (
+        "Expected all tensors in the given list to be XLA tensors. "
+        "Element at index 1 is not an XLA tensor. Got: torch.FloatTensor")
+    with self.assertRaisesRegex(RuntimeError, expected_regex=error_message):
+      torch_xla._XLAC._get_graph_hash(tensors)
```

Comment:
This is an internal API. Is it possible for a user to trigger this error via the public torch_xla API? If so, can we switch the test to use the public API so that it's more realistic?